engines/engine.h

```diff
@@ -52,42 +52,80 @@ class Dialog;
 }
 
 /**
- * Initializes graphics and shows error message.
+ * @defgroup engines_engine Engine
+ * @ingroup engines
+ *
+ * @brief API for managing various functions of the engine.
+ *
+ * @{
+ */
+
+/**
+ * Initialize graphics and show an error message.
+ * Redirects to GUIErrorMessage with U32Strings.
```

Comment:
You can drop the `Redirects to GUIErrorMessage with U32Strings.` part.
This was a comment for the one that takes a `Common::String` only, to indicates that it converts that string to a `Common::U32String` and then calls this overload. So having this comment for the overload that takes a `Common::U32String` does not make sense. And even for the other ones for which it could make sense, I don't think we need to have it in the documentation. This is more a comment about the way it is implemented.

Reply:
Just removing everywhere.